test/spec/modules/parrableIdSystem_spec.js

```diff
@@ -242,6 +247,133 @@ describe('Parrable ID System', function() {
         })
       });
     });
+
+    describe('third party cookie support', function () {
+      let logErrorStub;
+      let callbackSpy = sinon.spy();
+
+      beforeEach(function() {
+        logErrorStub = sinon.stub(utils, 'logError');
+      });
+
+      afterEach(function () {
+        callbackSpy.resetHistory();
+        removeParrableCookie();
+      });
+
+      afterEach(function() {
+        logErrorStub.restore();
+      });
+
+      describe('when getting tpcSupport from XHR response', function () {
+        let request;
+        let dateNowStub;
+        const dateNowMock = 1;
```

Comment:
Did we also want to compute a valid current date value here as well?
```suggestion
        const dateNowMock = Date.now();
```